front/app/modules/commercial/id_vienna_saml/components/ViennaSamlButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { AUTH_PATH } from 'containers/App/constants';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+import { StyledAuthProviderButton } from 'components/SignUpIn/AuthProviders';
+import { TOnContinueFunction } from 'components/SignUpIn/AuthProviderButton';
+
+// typings
+import { TSignUpInFlow } from 'components/SignUpIn';
+
+interface Props {
+  flow: TSignUpInFlow;
+  onContinue: TOnContinueFunction;
+}
+
+const ViennaSamlButton = ({ onContinue, flow }: Props) => {
+  const setHref = () => {
+    window.location.href = `${AUTH_PATH}/vienna_citizen`;
+  };
+  const handleOnContinue = () => {
+    if (flow === 'signup') {
+      window.location.replace(
```

Comment:
I could not find any other place in the code where we programmatically redirect to an external page like this. Is there a common way to do it instead?

Reply:
Replaced :+1: